integration-tests/api/__tests__/admin/sales-channels.js

```diff
@@ -614,60 +616,126 @@ medusaIntegrationTestRunner({
     })
 
     describe("DELETE /admin/sales-channels/:id/products/batch", () => {
+      // BREAKING CHANGE: Endpoint has changed
+      // from: DELETE /admin/sales-channels/:id/products/batch
+      // to: POST /admin/sales-channels/:id/products/batch/remove
+
       let salesChannel
       let product
 
       beforeEach(async () => {
-        product = await simpleProductFactory(dbConnection, {
-          id: "product_1",
-          title: "test title",
-        })
-        salesChannel = await simpleSalesChannelFactory(dbConnection, {
-          name: "test name",
-          description: "test description",
-          products: [product],
-        })
+        ;({ salesChannel, product } = await breaking(
+          async () => {
```

Comment:
No need to change anything now, but let's try to use the HTTP endpoints for seeding in the e2e  BE tests